Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -257,6 +259,46 @@ static unsigned typedArrayElementSize(ArrayBufferViewSubtag tag)
     }
 }
 
+static ErrorType errorNameToErrorTypeEnumValue(const String& name)
+{
+    if (equalLettersIgnoringASCIICase(name, "evalerror"_s))
+        return ErrorType::EvalError;
+    if (equalLettersIgnoringASCIICase(name, "rangeerror"_s))
+        return ErrorType::RangeError;
+    if (equalLettersIgnoringASCIICase(name, "referenceerror"_s))
+        return ErrorType::ReferenceError;
+    if (equalLettersIgnoringASCIICase(name, "syntaxerror"_s))
+        return ErrorType::SyntaxError;
+    if (equalLettersIgnoringASCIICase(name, "typeerror"_s))
+        return ErrorType::TypeError;
+    if (equalLettersIgnoringASCIICase(name, "urierror"_s))
+        return ErrorType::URIError;
+    return ErrorType::Error;
```

Comment:
We should define these enums separately in this SerializedScriptValue. `ErrorType` enum is not a defined interface in JSC. So this can be changed freely, and it can break serialization since it is relying on this errorType number serialized in persistent storage.